queries/scala/highlights.scm

```diff
@@ -0,0 +1,40 @@
+[
+  "abstract"
+  "case"
+  "catch"
+  "class"
+  "def"
+  "do"
+  "else"
+  "extends"
+  "final"
+  "finally"
+  "for"
+  "forSome"
+  "if"
```

Comment:
But the keywords should fit the subcategories we have in contributing.md

Reply:
you're right. okay I've made the changes I think you're looking for. lmk